docs/api/pet.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Pet
+nav_order: 3
+---
+
+# `pet` resource
+
+Base endpoint:
+
+```shell
+{base_url}/pets
+```
+
+Contains information about pets in the service.
+
+To adopt a pet in the service, the user must first be added to the service. Learn more about the [user resource](user.md).
+
+## Resource properties
+
+Sample `pet` resource:
+
+```js
+{
+    "id": 4,
+    "name": "Fred",
+    "type": "dog",
+    "breed": "Poodle",
+    "age": "2 years",
```

Comment:
Updated in https://github.com/syangabq/pet-adoption-service/pull/3

Reply:
Updated!